source/command-monitoring/tests/README.rst

```diff
@@ -50,6 +50,14 @@ The server is inconsistent on whether the ok values returned are integers or dou
 for simplicity the tests specify all expected values as doubles. Server 'ok' values of
 integers MUST be converted to doubles for comparison with the expected values.
 
+Batch Size
+^^^^^^^^^^
+
+On server versions 3.1 and higher, batch size when used in conjunction with limit is no
```

Comment:
Use 3.2 everywhere except the tests themselves. Later, we won't need to come back and change text as well.